templates/report.html.jinja

```diff
@@ -265,54 +282,52 @@
 
     <div class="responsive-prose max-w-none">
         <h2>
-            Consistency with the California GTFS Minimum Guidelines
+            Consistency with the California Transit Data Guidelines
         </h2>
-
+        <p>
+            The following checks measure adherence to the GTFS Schedule Compliance portion of the
+            <a href="https://dot.ca.gov/cal-itp/california-transit-data-guidelines"> California Transit Data Guidelines</a>.
+        </p>
+        <p>
+            The following symbols are used to denote each check's outcome on the given date:
+            <ul>
+                <li> <i class="fas fa-lg fa-check-circle text-itp-green"></i> Pass </li>
+                <li> <i class="fas fa-lg fa-ban text-itp-red/40"></i> Fail </li>
+                <li> <i class="far fa-lg fa-circle text-itp-slate-light"></i> Not applicable/data unavailable </li>
+            </ul>
+        </p>
+        <p>
+            Checks marked with an asterisk (*) are periodically performed manually. We can update these on request, so let us know if one of them looks off.
```

Comment:
Should `let us know` have a link to an appropriate email address where people could report this? (I also defer to @edasmalchi on copy)

Reply:
done!